web/client/plugins/Measure.jsx

```diff
@@ -175,7 +175,7 @@ const MeasurePlugin = connect(
     }))
 )(({coordsAeronauticalEnabled, ...props}) => {
     return (
-        <div className="measure-container" style={{top: coordsAeronauticalEnabled ? 88 : 48}}>
+        <div className={props.mapType !== MapLibraries.CESIUM ? "measure-container" : "measure-container measure-cesium"} style={{top: coordsAeronauticalEnabled ? 88 : 48}}>
```

Comment:
If you try to remove or set false the `showCoordinateEditor` property with this fix,
you get this layout issue

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/4a7ba725-0301-4283-a936-4f965b0f82dc)

The floating box for measure tools is valid both for 2D and 3D so it would be better to have a different class name instead of `meaure-cesium`, this are the case we have:

- showCoordinateEditor false 2D viewer has a floating toolbar
- showCoordinateEditor false 3D viewer has a floating toolbar
- showCoordinateEditor true 2D viewer has a side panel
- showCoordinateEditor true 3D viewer has a floating toolbar

please refactor the code taking into account these considerations